Client/Application/AppDelegate.swift

```diff
@@ -293,7 +217,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
         adjustIntegration?.triggerApplicationDidFinishLaunchingWithOptions(launchOptions)
 
         UNUserNotificationCenter.current().delegate = self
-        self.registerNotificationCategories()
+        SentTabAction.registerActions()
```

Comment:
Nit: use a singleton, rather than lots of static methods.

Reply:
Curious. I would think static methods would be better. There aren’t any resources we are creating/destroying with these static methods. I usually only like to use singletons in cases where we need to allocate a bunch of memory or hold a db connection and don’t want it to be created/destroyed every time.